corehq/apps/data_interfaces/tests/__init__.py

```diff
@@ -0,0 +1 @@
+from tests import *
```

Comment:
Relative imports of this style are not recommended. I think they will not work on Python3, for example. Recommended:
```python
from .test import *
```

I'm not really a fan of `import *` either, but not sure if the maintenance headache of updating explicit names is worth it. Our test runner should be smart enough to find the tests without this import dance! /rant (not your problem)